osu.Game/BackgroundDataStoreProcessor.cs

```diff
@@ -316,8 +316,7 @@ private void convertLegacyTotalScoreToStandardised()
 
             HashSet<Guid> scoreIds = realmAccess.Run(r => new HashSet<Guid>(r.All<ScoreInfo>()
                                                                              .Where(s => !s.BackgroundReprocessingFailed && s.BeatmapInfo != null
-                                                                                                                         && (s.TotalScoreVersion == 30000002
-                                                                                                                             || s.TotalScoreVersion == 30000003))
+                                                                                                                         && s.TotalScoreVersion < LegacyScoreEncoder.LATEST_VERSION)
```

Comment:
In my testing, which consisted of:

- importing the legacy replays from the issue thread on `master`
- moving to this branch

this change appears to be required to have the NaN score actually recompute correctly.

I'm not sure why this code previously insisted on listing every individual version separately, and I even [brought this up before](https://github.com/ppy/osu/pull/24685#discussion_r1311224281). I'm also thinking that the lack of changing this _might_ have had the effect of _no preexisting scores_ getting converted last release?

Reply:
From further testing, it seems that yes, users' pre-existing imported stable scores have actually not been recomputed on the latest release. See https://github.com/ppy/osu/pull/25876#discussion_r1431007020 for more  on this.

Reply:
Agree that this looks more correct.